src/amo/sagas/collections.js

```diff
@@ -1,3 +1,4 @@
+/* @flow */
```

Comment:
I ported this to Flow because Collections require a lot of complex actions and I can no longer keep it all in my head 😞 

This was a specific bug we had that Flow would have caught: https://github.com/mozilla/addons-frontend/issues/4323

When I converted it to Flow I noticed another bug! The `slug` parameter was incorrect.